syft/workers/websocket_client.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 
 
-TIMEOUT_INTERVAL = 9999999
+TIMEOUT_INTERVAL = 9_999_999
```

Comment:
Was this change intended?

Reply:
No , pretty surprising how it ended up.

Reply:
I think it was `black`.

Reply:
Yep , I guess let it remain that way.

Reply:
Apparently this is a valid pythonic way of writing an int (pep 515). Learned something new today

Reply:
Interesting :)